index.html

```diff
@@ -11,14 +11,79 @@
     />
   </head>
   <body class="app-grid">
-    <section class="header"></section>
 
-    <section class="content"></section>
+    <section class="header header__bar">
```

Comment:
This section element should be a `header`element. And the two classes can be merged into one.
```suggestion
    <header class="header">
```